code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -33,9 +33,10 @@
 			if(M.confused < drunk_value)
 				M.confused += 3
 		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1)
-	..()
-	return
+			M.adjustToxLoss(1, 0)
+			. = 1
+	return ..() | .
```

Comment:
oh, you're right, I was using `|` in previous version because I was using `return . | ..()` (and we always need to call the "..()" ). Will change.